lmdeploy/lite/quantization/awq.py

```diff
@@ -228,10 +246,10 @@ def smooth_fc_fcs(pre_fc: torch.nn.Module,
         fc.weight.mul_(scales.view(1, -1))
 
     for p in pre_fc.parameters():
-        assert torch.isnan(p).sum() == 0
+        assert torch.isnan(p).sum() == 0, 'may use `--clamp-zeros` option'
```

Comment:
Is the 0 divisor when calculating scale the only reason to NAN?

Reply:
Yes, the only reason.